### PR TITLE
User scenario for changing user password

### PR DESCRIPTION
**Pull request by leebrandt**
Closes: #83

### CODE REVIEW

File: src/Okta.Sdk.IntegrationTests/UserScenarios.cs [hunk 1]
```diff
@@ -205,5 +205,47 @@ public class UserScenarios : ScenarioGroup
...
+
+            try
+            {
+                var updatedUserCredentials = await client.Users.ChangePasswordAsync(
+                    new ChangePasswordRequest
+                    {
+                        OldPassword = new PasswordCredential { Value = "Abcd1234" },
+                        NewPassword = new PasswordCredential { Value = "1234Abcd" },
+                    },
+                    createdUser.Id);
```
Comment (nbarbettini) [inline src/Okta.Sdk.IntegrationTests/UserScenarios.cs]: Add a TODO here to remind us to make this prettier.

**Comment by nbarbettini** [comment]
Can you rebase this to master as well? Should take care of the merge conflict.

File: src/Okta.Sdk.IntegrationTests/UserScenarios.cs (hunk 1 above)
Comment (leebrandt) [inline src/Okta.Sdk.IntegrationTests/UserScenarios.cs]: In like FLynn

Comment (leebrandt) [inline src/Okta.Sdk.IntegrationTests/UserScenarios.cs]: :boom:

**Comment by nbarbettini** [comment]
Resolved conflicts, rebased, and squashed. Feel free to pull it down to make sure I didn't screw anything up, and then 🚀 merge it @leebrandt!

**Pull request merged**